OSXCore/HangulComposer.swift

```diff
@@ -192,6 +198,11 @@ final class HangulComposer: NSObject, Composer {
     {
         // libhangul은 backspace를 키로 받지 않고 별도로 처리한다.
         if keyCode == .delete {
+            // JDK 호환 모드의 경우, _compatiblePreeditString가 있을 경우 그 문자를 우선적으로 지운다
+            if configuration.hangulJDKCompatible, _compatiblePreeditString.isEmpty == false {
+                _compatiblePreeditString = ""
```

Comment:
이렇게 구현되려면 이 값이 String이 아니고 Character여야 하겠네요.

안써봤는데 dropLast가 있다고 해요
```suggestion
                _compatiblePreeditString.dropLast()
```